uaclient/cli/cli_util.py

```diff
@@ -6,8 +6,9 @@
 
 
 class CLIEnableDisableProgress(api.AbstractProgress):
-    def __init__(self):
-        self.is_interactive = True
+    def __init__(self, *, assume_yes: bool):
+        self.is_interactive = not assume_yes
+        self.assume_yes = assume_yes
```

Comment:
Why do we need both variables here ? Can't we just use `is_interactive` ?

Reply:
`progress.is_interactive()` is used for different purposes than `assume_yes` and needs to work on any progress object, even ones that don't have `assume_yes`. So this implementation of `is_interactive` is trivial, but it is useful to keep the two concepts separated.